src/Simplify.cpp

```diff
@@ -40,6 +40,12 @@ Simplify::Simplify(bool r, const Scope<Interval> *bi, const Scope<ModulusRemaind
 
     use_synthesized_rules = get_use_synthesized_rules_from_environment();
 
+    // Don't require this one to be set
+    if (get_env_variable("HL_DISABLE_CAN_PROVE_RULES") == "1") {
```

Comment:
Simplify instances are constructed somewhat frequently, so I'm a little worried this might hurt compile times, which we are benchmarking. Not sure how fast an env var lookup is. Can this be made static?

Reply:
Ah, yes: I didn't notice the static usage for HL_USE_SYNTHESIZED_RULES.